packages/suite/src/support/messages.ts

```diff
@@ -2642,6 +2675,10 @@ const definedMessages = defineMessages({
         defaultMessage: 'Testnet coins',
         id: 'TR_TESTNET_COINS',
     },
+    TR_TESTNET_COINS_LOWERCASE: {
```

Comment:
Why maintain two translation strings if the one above could be used with `.toLowerCase()`.

Reply:
yeah, looks like we would be fine with one string
cc @tomexx

Reply:
Sorry, good spot! @keraf